apis/core/v1beta1/featureflaginprocessconfiguration_types.go

```diff
@@ -0,0 +1,218 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"fmt"
+
+	"github.com/open-feature/open-feature-operator/apis/core/v1beta1/common"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// FeatureFlagInProcessConfigurationSpec defines the desired state of FeatureFlagInProcessConfiguration
+type FeatureFlagInProcessConfigurationSpec struct {
```

Comment:
@toddbaert related to your review comment [1]

what if we shorten the name to `InProcessConfiguration` 🤔  we don't need `FeatureFlag` prefix as this is simply configurations.  

[1] - https://github.com/open-feature/open-feature-operator/pull/632#pullrequestreview-2072356217

Reply:
I think `InProcessConfiguration` is slightly better, but I don't feel very strongly about it.

@Kavindu-Dodan @beeme1mr @thisthat I think this is the only remaining concern for me (this name). If we can't think of anything better I am in favor of merging.

Reply:
adapted to `InProcessConfiguration`

Reply:
Personally I like `InProcessConfiguration ` as it communicate clearly the purpose of the crd. 

@toddbaert @thisthat @beeme1mr  over to you for final approval :)